src/cleed++/Util/teebuf.hh

```
@@ -0,0 +1,56 @@
+    else
+    {
+      char_type const ch = traits::to_char_type(c);
+      int_type const r1 = sb1->sputc(ch);
+      int_type const r2 = sb2->sputc(ch);
+      
+      return (traits::eq_int_type(r1, eof) ||
+			  traits::eq_int_type(r2, eof) ) ? eof : c;
+    }
+  }
+  
+private:
```

Clang (14.0.6)

[warning] redundant access specifier has the same accessibility as the previous access specifier

(readability-redundant-access-specifiers)